### DISCUSSION TITLE
Need details on Valkey Metrics

### DISCUSSION THREAD

**Discussion by vinuthana12** [valkey-io/valkey#1352] [Q&A]
Need information on 
1) Does valkey supports prometheus format metrics
2) If so need details about which port and path it's exposed, 

Please also help us with all the required docs for this

**Comment by zioproto** [comment]
It seems the current implementation of Redis / Valkey do not implement a Prometheus scraping point. It seems a lot of people are using the project [redis_exporter](https://github.com/oliver006/redis_exporter) to pull metrics out of Valkey.

It would be interesting to know more about the plans for a Prometheus instrumentation that does not require to run an external exporter.
